Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.7-alpine
```

Comment:
feedparser dependency isn't compatible with Python 3.7. It should be 3.6 for now.

Reply:
Will fix.